test

```diff
@@ -11,6 +11,9 @@ LINTABLE=$( go list -tags=golint -f '
 
 go test -v -i -race github.com/coreos/go-oidc/...
```

Comment:
Remove the first one?

Reply:
I assume it had something to do with the `-i` flag, which is deprecated. I would just remove that one.

Reply:
Done, thank you!